src/extension.ts

```diff
@@ -274,20 +274,20 @@ function activateIoTHub(context: vscode.ExtensionContext, outputChannel: vscode.
 
     const azureIoTExplorer = new AzureIoTExplorer(outputChannel, context, iotHubExtTreeDataProvider);
 
-    context.subscriptions.push(iotHubTreeItem);
-    context.subscriptions.push(vscode.window.createTreeView("iotHubExplorer", { treeDataProvider: iotHubExtTreeDataProvider, showCollapseAll: true }));
+    // context.subscriptions.push(iotHubTreeItem);
+    // context.subscriptions.push(vscode.window.createTreeView("iotHubExplorer", { treeDataProvider: iotHubExtTreeDataProvider, showCollapseAll: true }));
 
-    registerCommand("azure-iot-hub.setIoTHub", async (actionContext: IActionContext, node?: IoTHubResourceTreeItem) => {
-        await azureIoTExplorer.setIoTHub(actionContext, node);
-    });
+    // registerCommand("azure-iot-hub.setIoTHub", async (actionContext: IActionContext, node?: IoTHubResourceTreeItem) => {
```

Comment:
these commands descriptions are still in package.json right?

Reply:
Yes, but there would have no impact.

Reply:
If not registered better to comment out the description. Besides we may add some comment or TODO tag here for maintenance.

Reply:
Besides, I think for internal command. They could just be removed from package.json right? Like what we did in the module marketplace.

Reply:
Updated.